src/optimizer/filter_combiner.cpp

```diff
@@ -451,6 +460,9 @@ bool FilterCombiner::IsDenseRange(vector<Value> &in_list) {
 	// check if the gap between each value is exactly one
 	hugeint_t prev_value = in_list[0].GetValue<hugeint_t>();
 	for (idx_t i = 1; i < in_list.size(); i++) {
+		if (in_list[i].IsNull()) {
+			return true;
```

Comment:
This should probably return false

Reply:
The idea was to return true to tricking the join filter pushdown into not creating the dynamic filter. If the range is dense then the `>= and <=` filter will do all the work. But I didn't like the logic flow, and it also omits cases where the values are varchars, and then `IsDenseRange` automatically returns True